### PR TITLE
test to check if node group has single VIP IP

### PR DESCRIPTION
**Pull request by vkhalin**
test that checks do we have duplicates for VIP IP in each node group

### CODE REVIEW

File: mk_verificator/tests/system/single_vip/test_single_vip.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    local_salt_client.cmd(group, 'saltutil.sync_all')
+    nodes_list = local_salt_client.cmd(group, 'grains.item', ['ipv4'])
+
+    ipv4_list = []
+
+    for node in nodes_list:
+        ipv4_list.extend(nodes_list.get(node).get('ipv4'))
+
+    ipv4_list.sort()
+    ipv4_list = list(filter(lambda ip: ip != '127.0.0.1', ipv4_list))
```
Comment (msenin94) [inline mk_verificator/tests/system/single_vip/test_single_vip.py]: I would suggest to use Counter for calculations: https://docs.python.org/2/library/collections.html

Comment (vkhalin) [inline mk_verificator/tests/system/single_vip/test_single_vip.py]: @msenin94 I will rewrite this test in next commit to use Counter. Is that ok?

Comment (msenin94) [inline mk_verificator/tests/system/single_vip/test_single_vip.py]: let's keep this one and finalize, that's not critical for other stuff to merge it right now ;-)

**Comment by msenin94** [comment]
good for me now

**Pull request merged**